src/obs-ndi-source.cpp

```diff
@@ -573,6 +563,19 @@ void *ndi_source_thread(void *data)
 				break;
 			}
 
+			// Force a clean (blank) frame when settings change to Audio only
+			if (recv_desc.bandwidth ==
+			    NDIlib_recv_bandwidth_audio_only) {
+				obs_source_output_video(obs_source,
```

Comment:
Per https://docs.obsproject.com/reference-sources#c.obs_source_output_video
Just use:
```
obs_source_output_video(obs_source, nullptr);
```

Reply:
Will update everywhere and get rid of the config.blank_frame then